INSTALL

```diff
@@ -68,8 +68,7 @@ you may need to install the development packages (look for a "-dev"
 postfix) in addition to the core packages.  
 
 python 2.7 or later (http://www.python.org/)
-    Iris requires Python 2.7 or later, but is not currently compatible with
-    Python 3.
+    Iris requires Python 2.7 or later.
```

Comment:
Is this technically true? We only test against 2.7 and 3.4, and I thought there was some reason for stating we only support these versions (@rhattersley, @qulogic?). In any case I think this needs to be more specific so users immediately realise we they can use Iris on Python 3.